bundle/manifests/instana-agent-operator.clusterserviceversion.yaml

```diff
@@ -326,6 +332,30 @@ spec:
           - securitycontextconstraints
           verbs:
           - use
+        - apiGroups:
+          - autoscaling/v1
+          resources:
+          - horizontalpodautoscalers
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - autoscaling/v2
+          resources:
+          - horizontalpodautoscalers
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - pods/log
+          verbs:
+          - get
+          - list
+          - watch
```

Comment:
**question:** Is there a reason why these have changed?

Reply:
They are already used on the ClusterRole in 
https://github.com/instana/instana-agent-operator/blob/8f444abf0fe2db67f9c513958f76d43effb97536/config/rbac/role.yaml#L175-L182 and https://github.com/instana/instana-agent-operator/blob/8f444abf0fe2db67f9c513958f76d43effb97536/config/rbac/role.yaml#L191-L198